zulipterminal/ui_tools/boxes.py

```diff
@@ -133,9 +138,10 @@ def stream_box_view(self, stream_id: int, caption: str='', title: str='',
 
     def stream_box_edit_view(self, stream_id: int, caption: str='',
                              title: str='') -> None:
-        self.stream_box_view(stream_id, caption, title)
+        self.stream_box_view(stream_id, caption, title, True)
```

Comment:
The boolean being passed by itself does not provide much context on why we're using it, you could perhaps use `*,` to require the parameter's name to be specified along with the value. This should provide context to understand the need for the boolean.

Reply:
Yes that seems to be a better idea, I would you keyword arguments for this purpose. Thanks for pointing this out.